src/main/java/core/basesyntax/service/impl/CalculateDataImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.CalculateData;
+import core.basesyntax.strategy.FruitStrategyImpl;
+import java.util.List;
+import java.util.Map;
+
+public class CalculateDataImpl implements CalculateData {
+    static final int OPERATION_INDEX = 0;

```

Comment:
```suggestion
    private static final int OPERATION_INDEX = 0;

```
add private static final everywhere